ipi_tests/regression_tests/tests/DRIVEN/test_settings.dat

```diff
@@ -0,0 +1,4 @@
+driver_model zundel  
+address localhost
+port 32342 
+socket_mode unix
```

Comment:
Commenting here for the whole regtest - wouldn't it be better to call it "dmd" rather than "DRIVEN" so that the connection to the code is more apparent?